core-test/src/main/java/org/openstack4j/api/identity/v3/KeystoneTokenlessTest.java

```diff
@@ -0,0 +1,49 @@
+package org.openstack4j.api.identity.v3;
+
+import okhttp3.mockwebserver.RecordedRequest;
+import org.openstack4j.api.AbstractTest;
+import org.openstack4j.api.OSClient;
+import org.openstack4j.openstack.OSFactory;
+import org.openstack4j.openstack.identity.v3.domain.KeystoneToken;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Tests the Identity/Keystone API version 3 Keystone tokenless
+ */
+@Test(groups = "idV3", suiteName = "Identity/V3/Keystone")
+public class KeystoneTokenlessTest extends AbstractTest {
+
+    private static final String JSON_USERS = "/identity/v3/users.json";
+
+    /**
+     * check headers whether right from request
+     *
+     * @throws Exception
+     */
+    public void pass_headers_Test() throws Exception {
+
+        respondWith(JSON_USERS);
+
+        Map header = new HashMap();
+        String key = "X-Domain-Id";
```

Comment:
* domainScope:
X-Domain-Id:If specified, its the domain scope.
* projectScope:
X-Project-Id: If specified, its the project scope.
X-Project-Domain-Id:If specified, its the domain of project scope.

here, i test it X-Domain-Id for domainScope.
please refer to:https://docs.openstack.org/developer/keystone/configure_tokenless_x509.html